repositories/rust/rustfmt/generate.sh

```diff
@@ -2,4 +2,7 @@
 
 set -o errexit -o nounset
 
-bazel run @rules_rust//tools/upstream_wrapper:rustfmt --script_path="$(dirname "$0")/rustfmt_wrapper_impl.sh"
+bazel run \
+    --noenable_bzlmod --enable_workspace \
```

Comment:
Why is this change needed? Bazel should pick up bazelrc config.

Reply:
It's from the Bzlmod PR but I thought I'd sneak it in here too as it has no adverse effects when there's no Bzlmod yet. The issue is that the script doesn't work with Bzlmod (but also it doesn't need to)